draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -624,6 +624,30 @@ This practice causes increased complexity in the client application and increase
 RADIUS/DTLS clients SHOULD implement session resumption, preferably stateless session resumption as given in {{RFC5077}}.
 This practice lowers the time and effort required to start a DTLS session with a server and increases network responsiveness.
 
+# Implementation Guidelines
+
+The text above describes the protocol. In this section, we give additional implementation guidelines. These guidelines are not part of the protocol, but they may help implementors create simple, secure, and interoperable implementations.
+
+## Client Implementations
+
+RADIUS/(D)TLS clients should use connected sockets where possible. Use of connected sockets means that the underlying kernel tracks the sessions so that the client subsystem does not need to manage multiple sessions on one socket.
+
+RADIUS/(D)TLS clients should use a single source (IP + port) when sending packets to a particular RADIUS/(D)TLS server. Doing so minimizes the number of (D)TLS session setups. It also ensures that information about the home server state is discovered only once.
```

Comment:
This could be a problem if a single RADIUS/(D)TLS peer has so much packet throughput that a single connection would be clogged (due to the limited ID space), and in this case the client would have to open a second connection to the same server to increase the usable space for packets-in-flight.

I think we don't have text for that in the document yet, maybe it should be included in #13